Formula/sslscan.rb

```diff
@@ -17,10 +17,12 @@ class Sslscan < Formula
 
   def install
     system "make"
+    mkdir_p [bin, man1]
     system "make", "install", "PREFIX=#{prefix}"
   end
 
   test do
+    assert_match version.to_s, shell_output("#{bin}/sslscan --version")
```

Comment:
@ilovezfs i'll second @DomT4's idea that we don't need to recommend these; this is fine here though!